.github/workflows/master.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [12.x, 14.x]
```

Comment:
let's perhaps remove matrix altogether until we have this open-source. just so we can preserve a little build minutes, and open-source we get free-for-all, then we can all ALL valid node versions there in matrix :trollface:

Reply:
This is open source leo!

Reply:
changed to [12.x, 14.x, 16.x] in squashed commit